### PR TITLE
feat: update apollox token list

### PR DESCRIPTION
**Pull request by AQVYmeCkJsFTFWpMAHmA**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. The protocol is usually listed within 24 hours of merging the PR
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/apollox exports TVL: 
        
 
 
    bsc-staking               12.30 M
    bsc                       28.98 M
    bsc-pool2                 4.07 M
    staking                   12.30 M
    pool2                     4.07 M
    
    total                    28.98 M

**Comment by g1nt0ki** [comment]
@AQVYmeCkJsFTFWpMAHmA thanks for the PR, I have not merged it yet as it needs some changes, like arbi and eth treasury needs to added to different chain, also I had disabled apollox token because it might be used to inflate project tvl

**Comment by g1nt0ki** [comment]
I will update this adapter soon

File: projects/apollox/index.js [hunk 1]
```diff
@@ -15,14 +19,14 @@ const TOKEN_BUSD = "0xe9e7CEA3DedcA5984780Bafc599bD69ADd087D56";
 const TOKEN_CAKE = "0x0e09fabb73bd3ade0a17ecc321fd13a19e81ce82";
 const TOKEN_BABY = "0x53e562b9b7e5e94b81f10e96ee70ad06df3d2657";
 const TOKEN_LEOS = "0x2c8368f8f474ed9af49b87eac77061beb986c2f1";
-const TOKEN_USDC = "0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d";
+const TOKEN_USDC = "0x8ac76a51cc950d982268b83fe1ad97b32cd580d";
```
Comment (g1nt0ki) [inline projects/apollox/index.js:22]: think these fixes are incorrect, USDC token points to nothing and TOKEN_MDX is invalid: https://www.bscscan.com/search?f=0&q=0x8ac76a51cc950d982268b83fe1ad97b32cd580d

**Comment by g1nt0ki** [comment]
closing this in favor of https://github.com/DefiLlama/DefiLlama-Adapters/commit/5a83d7b94b609e75b98c13f6c6e67e686c1d2650

**Pull request closed without merging**